src/ci/citool/src/merge_report.rs

```diff
@@ -266,18 +282,34 @@ fn report_test_diffs(mut diff: AggregatedTestDiffs) {
     grouped_diffs.sort_by(|(d1, g1), (d2, g2)| g1.cmp(&g2).then(d1.test.name.cmp(&d2.test.name)));
 
     for (diff, job_group) in grouped_diffs {
-        println!("- `{}`: {} (*J{job_group}*)", diff.test.name, format_diff(&diff.diff));
+        println!(
+            "- `{}`: {} ({})",
+            diff.test.name,
+            format_diff(&diff.diff),
+            format_job_group(job_group)
+        );
     }
 
-    let extra_diffs = diff.diffs.len().saturating_sub(max_diff_count);
+    let extra_diffs = diffs.len().saturating_sub(max_diff_count);
     if extra_diffs > 0 {
         println!("\n(and {extra_diffs} additional {})", pluralize("test diff", extra_diffs));
     }
 
-    // Now print the job index
-    println!("\n**Job index**\n");
+    if doctest_count > 0 {
+        println!(
+            "\nAdditionally, {doctest_count} doctest {} were found.",
+            pluralize("diff", doctest_count)
+        );
+    }
```

Comment:
Suggestion: maybe explicitly remark that doctest diffs are not reported due to being too noisy?

Reply:
I feel like there is already enough text (and there will be more =D), but it's true that this should be explained.